test/TestUtilities/Test.Utility/TestFileSystemUtility.cs

```diff
@@ -146,5 +187,24 @@ public static IDisposable SetCurrentDirectory(string path)
                 OldPath = oldPath
             };
         }
+
+        public static IEnumerable<DirectoryInfo> ParentDirectoryLookup()
+        {
+            var currentDirInfo = new DirectoryInfo(Directory.GetCurrentDirectory());
+            while (currentDirInfo != null)
+            {
+                yield return currentDirInfo;
+                currentDirInfo = currentDirInfo.Parent;
+            }
+
+            yield break;
+        }
+
+        public static bool DirectoryContains(DirectoryInfo directoryInfo, string subDirectory)
+        {
+            return directoryInfo
+                .EnumerateDirectories()
+                .Any(dir => StringComparer.OrdinalIgnoreCase.Equals(dir.Name, subDirectory));
+        }
```

Comment:
`Directory.Exists(Path.Combine(directoryInfo.Name, subDirectory))` should work, and avoids having to enumerate the directory. While NuGet has a huge problem of methods that are far too long, personally I'm not sure if I think a helper function for `Directory.Exists` is very useful. Particularly since `DirectoryContains` sounds like it should be checking files as well.

Reply:
Thanks for the suggestion! Addressed accordingly.